project.clj

```diff
@@ -4,9 +4,12 @@
   :license {:name "EPL-2.0 OR GPL-2.0-or-later WITH Classpath-exception-2.0"
             :url  "https://www.eclipse.org/legal/epl-2.0/"}
   :dependencies [[org.clojure/clojure "1.10.0" :scope "provided"]
-                 [org.slf4j/slf4j-log4j12 "1.7.32"]
+                 [com.fzakaria/slf4j-timbre "0.3.21"]
                  [com.taoensso/timbre "5.1.0"]
                  [de.active-group/active-clojure "0.40.0"]
                  [de.active-group/timbre-riemann "0.2.0"]
                  [cheshire "5.10.1"] ;; for logstash 3rd-party-appender of timbre
-                 [riemann-clojure-client "0.5.1" :exclusions [org.slf4j/slf4j-api]]])
+                 [riemann-clojure-client "0.5.1" :exclusions [org.slf4j/slf4j-api]]
+                 [org.slf4j/log4j-over-slf4j "1.7.36"]
+                 [org.slf4j/jul-to-slf4j     "1.7.36"]
+                 [org.slf4j/jcl-over-slf4j   "1.7.36"]])
```

Comment:
We also have `deps.edn` in this project that should be updated as well -- is there an automatic way to keep `project.clj` and `deps.edn` in sync?

Reply:
I think this is an artifact from when we started building the library and later moved to `leiningen`.  Since it is currently unused as well as out of sync with `project.clj`, I've deleted it for the time being. See 5962f67c3ce0cbe997b31053970e761b59d99223